src/userActivity.js

```diff
@@ -61,9 +61,17 @@ class UserActivity {
   }
 
   //Below: For a user, find their all-time stair climbing record
-  // findAllTimeStairRecord() {
-  //
-  // }
+  findAllTimeStairRecord() {
+    let findStairsRecords = this.userActivityInformation.map(day => day.flightsOfStairs);
+    let allTimeStairsRecord = this.userActivityInformation.find(day => {
+      if (day.flightsOfStairs === Math.max(...findStairsRecords))
+        return day;
```

Comment:
I know we were going to look for a more concise way to do this, but hey, it works now and doesn't look too bad